MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import UIKit
+
+struct AlertPresenter {
+    weak var viewController: UIViewController?
+    
+    func showAlert (alertModel:AlertModel) {
```

Comment:
Перед круглой скобкой обычно пробел не ставят, 
плюс ставьте пробел между именем и объявлением типа:
```suggestion
    func showAlert(alertModel: AlertModel) {
```